jsapp/js/account/usage/usage.api.ts

```diff
@@ -73,3 +72,40 @@ export async function getUsage(organization_id: string | null = null) {
     errorMessageDisplay: t('There was an error fetching usage data.'),
   });
 }
+
+export async function getAssetUsage(url?: string) {
+  const apiUrl = url || ASSET_USAGE_URL;
+  return fetchGet<AssetUsage>(apiUrl, {
+    includeHeaders: true,
+    errorMessageDisplay: t('There was an error fetching asset usage data.'),
+  });
+}
+
+export async function getAssetUsageForOrganization(
+  pageNumber: number,
+  order?: ProjectsTableOrder
+) {
+  let organizations;
+  try {
+    organizations = await getOrganization();
+  } catch (error) {
+    return await getAssetUsage(ASSET_USAGE_URL);
+  }
+
+  let apiUrl = ORGANIZATION_ASSET_USAGE_URL.replace(
+    '##ORGANIZATION_ID##',
+    organizations.results?.[0].id || ''
+  ).replace('##PAGE_NUM##', pageNumber.toString());
+
+  if (
+    order?.fieldName &&
+    order.direction &&
+    (order.direction === 'ascending' || order.direction === 'descending')
+  ) {
+    const orderingPrefix = order.direction === 'ascending' ? '' : '-';
+    const fieldDefinition = PROJECT_FIELDS[order.fieldName as ProjectFieldName];
+    apiUrl += `&ordering=${orderingPrefix}${fieldDefinition.apiOrderingName}`;
```

Comment:
We're inconsistent on how to generate query param strings. I see some usage of URLSearchParams. I think a strategy that puts them in a map and then converts to a string at the end is cleaner.